infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/SidecarServicesProvisioner.java

```diff
@@ -67,6 +71,10 @@ public void provision(KubernetesEnvironment kubernetesEnvironment)
     }
   }
 
+  private boolean isDiscoverable(ChePluginEndpoint endpoint) {
+    return Boolean.parseBoolean(endpoint.getAttributes().getOrDefault("discoverable", "true"));
```

Comment:
It would be nice to add the corresponding constant to [ServerConfig](https://github.com/eclipse/che/blob/master/core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/ServerConfig.java) class.
Sorry for a late review.